app/react/edge/edge-stacks/CreateView/InnerForm.tsx

```diff
@@ -124,7 +124,11 @@ export function InnerForm({
       <StaggerFieldset
         isEdit={false}
         values={values.staggerConfig}
-        onChange={(value) => setFieldValue('staggerConfig', value)}
+        onChange={(value) =>
```

Comment:
replace `setFieldValue` with `setValues` 

```ts
onChange={newStaggerValues => 
   setValues(values => ({
				...values,
        stagerConfig: {
          ...values.staggerConfig,
          ...newStaggerValues
       }
   })
}
```

Reply:
object.entries is a nice solution, but formik has support for this. calling `setFieldValue` doesn't stack, so we will get a few rerenders